raster/r.report/testsuite/test_r_report.py

```diff
@@ -58,6 +64,613 @@ def test_output(self):
+                    ],
+                    "categories": [
+                        {
+                            "category": 27603,
+                            "label": "RALEIGH",
+                            "units": [
+                                {"unit": "cell counts", "value": 429179},
+                                {"unit": "% cover", "value": 34.959906551800536},
+                            ],
+                        },
+                        {
+                            "category": 27606,
+                            "label": "RALEIGH",
+                            "units": [
+                                {"unit": "cell counts", "value": 662642},
+                                {"unit": "% cover", "value": 53.977250511553954},
+                            ],
+                        },
+                        {
+                            "category": 27610,
+                            "label": "RALEIGH",
+                            "units": [
+                                {"unit": "cell counts", "value": 135811},
+                                {"unit": "% cover", "value": 11.062842936645509},
+                            ],
+                        },
+                    ],
+                },
+                {
+                    "category": 6,
+                    "label": "RALEIGH-WEST",
+                    "units": [
+                        {"unit": "cell counts", "value": 208989},
+                        {"unit": "% cover", "value": 10.320444444444444},
+                    ],
+                    "categories": [
+                        {
+                            "category": 27607,
+                            "label": "RALEIGH",
+                            "units": [
+                                {"unit": "cell counts", "value": 208989},
+                                {"unit": "% cover", "value": 100},
+                            ],
+                        }
+                    ],
+                },
+            ],
+            "totals": [
+                {"unit": "cell counts", "value": 2025000},
+                {"unit": "% cover", "value": 100},
+            ],
+        }
+        module = SimpleModule("r.report", map="towns,zipcodes", format="json")
+        self.runModule(module)
+        data = json.loads(module.outputs.stdout)
+        self._assert_report_equal(reference, data)
+
+    def test_json2(self):
+        """Test JSON format with more options"""
+        reference = {
+            "location": "nc_spm_full_v2alpha2",
+            "created": "2024-07-24T14:59:09+0530",
+            "region": {
+                "north": 228500,
+                "south": 215000,
+                "east": 645000,
+                "west": 630000,
+                "ew_res": 10,
+                "ns_res": 10,
+            },
+            "mask": None,
+            "maps": [
+                {
+                    "name": "towns",
+                    "description": "PERMANENT",
```

Comment:
Is PERMANENT really the description in the dataset? It happens to be like the common name used for mapset in the GRASS database (data folder structure).

Reply:
Fixed the description field and also renamed it to label to be consistent with the rest of the output.

Reply:
Yep, I saw it was named label and outputs the label. I assume that meant that the mapset isn't important to output.

Reply:
I am not sure if its relevant but I can add another field for it here if desired.

Reply:
I'm not the one deciding if it is relevant or not for these conceptual questions it was just my observation. I'm not asking anything on it (yet).

Reply:
@cwhite911 any suggestions?

Reply:
@echoix what do you think about merging this PR for now and we can add more fields in future if necessary?

Reply:
Yep makes sense, adding isn't problematic.